research-software-directory/RSD-as-a-service
# 1212 Categories - suggested layout and structure changes

# Suggested changes

Changes proposed in this pull request:
* Layout and visual approach changes for adding/editing categories
* Restructuring and the location of some files (follow approach in other "modules")
* No highlights option in custom community categories
* Shortened community settings menu item subtitles

How to test:
*  enable communities module in `public/data/settings.json`
* `make start` to build and create test data
* login as rsd admin
* create/edit/delete general categories in admin section
* create/edit/delete categories in the communities section
* check communities settings subtitles

## Categories (top) example

![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/71cab972-2d0f-4076-8397-91dfff12afa5)

## Categories  add child

![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/6bbdaf05-0a88-4b5a-96d6-f48c226531c9)

## Categories edit item

![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/ac8b1580-3eaa-4901-951b-54577dee3b16)


PR Checklist:
